geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -771,6 +771,8 @@ public synchronized String dump() {
 
   private boolean regionCreationNotified;
 
+  private boolean sentGWStopped = false;
```

Comment:
Could this (and the accessor methods for it) be renamed to something a little clearer? Maybe "sentGatewaySenderStoppedMessage"?

Reply:
updated